website/package.json

```diff
@@ -114,8 +112,10 @@
     "jest-environment-jsdom": "^29.5.0",
     "msw": "^0.49.3",
     "msw-storybook-addon": "^1.8.0",
+    "path-browserify": "^1.0.1",
```

Comment:
We already use this in `website/.storybook/main.js` so it makes sense to explicitly add this to the dev dependencies.